Xamarin.Forms.Platform.iOS/CollectionView/VerticalDefaultCell.cs

```diff
@@ -3,12 +3,12 @@
 
 namespace Xamarin.Forms.Platform.iOS
 {
-	internal sealed class DefaultVerticalListCell : DefaultCell
+	internal sealed class VerticalDefaultCell : DefaultCell
 	{
 		public static NSString ReuseId = new NSString("Xamarin.Forms.Platform.iOS.DefaultVerticalListCell");
```

Comment:
It might be good to rename this to match the type name - at least until we go stable.